modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -129,19 +136,29 @@ public void setDownloadurl(String downloadurl) {
         this.downloadurl = downloadurl;
     }
 
-
     @JsonIgnore
-    public Map<String, String> getCoordinates() {
+    public Map<String, String> getPurls() {
         return externalIds.entrySet().stream()
-                .filter(e -> SW360CoordinateKeysToArtifactCoordinates.getValues().contains(e.getKey()))
+                .filter(e ->PURL_TYPE.stream()
+                .anyMatch(purlType -> e.getKey().equals(purlType)))
                 .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
     }
 
-    public SW360Release setCoordinates(Map<String, String> coordinates) {
-        externalIds.putAll(coordinates);
+    public SW360Release setPurls(Map<String, String> purls) {
+        dropAllPurls();
+
+        purls.forEach(externalIds::put);
+
         return this;
     }
 
+    private void dropAllPurls() {
+        externalIds.keySet().stream()
+                .filter(e -> PURL_TYPE.stream()
+                        .anyMatch(e::startsWith))
```

Comment:
Why startsWith? Isn't that redundant code to my comment above, should then be a method, I think